lib/bridge.js

```diff
@@ -607,6 +633,19 @@ Bridge.prototype.provisionUser = function(matrixUser, provisionedUser) {
             return newUser.setAvatarUrl(provisionedUser.url);
         });
     }
+    if (!provisionedUser.dontJoinGroup && self.groupId) {
```

Comment:
I think it'd be better to use a positively-named boolean (something like shouldJoinGroup) that defaults to true.